internal.nix

```diff
@@ -283,6 +285,22 @@ rec {
     else
       throw "sourceHashFunc: spec.type '${spec.type}' is not supported. Supported types: 'github'";
 
+  runInstallScriptsForRootPackage = ''
```

Comment:
we could move this to a bash function, similar to `buildPhase` in `stdenv.mkDerivation`